configuration/parameters.js

```diff
@@ -0,0 +1,241 @@
+      baseUnit: "1000000",
+      priceSource: "1",
+      fixedPrice: "1000000",
+      uniswapMarket: "0x0000000000000000000000000000000000000000",
+      reporter: "0x0000000000000000000000000000000000000000",
+      reporterMultiplier: "1",
+      isUniswapReversed: false
+    },
+    {
+      // "NAME": "USDT",
+      cToken: "0xf650C3d88D12dB855b8bf7D11Be6C55A4e07dCC9",
+      underlying: "0xdAC17F958D2ee523a2206206994597C13D831ec7",
+      symbolHash: "0x8b1a1d9c2b109e527c9134b25b1a1833b16b6594f92daa9f6d9b7a6024bce9d0",
+      baseUnit: "1000000",
+      priceSource: "1",
+      fixedPrice: "1000000",
+      uniswapMarket: "0x0000000000000000000000000000000000000000",
+      reporter: "0x0000000000000000000000000000000000000000",
+      reporterMultiplier: "1",
+      isUniswapReversed: false
+    },
+    {
+      // "NAME": "WBTCv2",
+      cToken: "0xccf4429db6322d5c611ee964527d42e5d685dd6a",
+      underlying: "0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599",
+      symbolHash: "0xe98e2830be1a7e4156d656a7505e65d08c67660dc618072422e9c78053c261e9",
+      baseUnit: "100000000",
+      priceSource: "2",
+      fixedPrice: "0",
+      uniswapMarket: "0xcbcdf9626bc03e24f779434178a73a0b4bad62ed",
+      reporter: "0x4846efc15CC725456597044e6267ad0b3B51353E",
+      reporterMultiplier: "1000000",
+      isUniswapReversed: false
+    },
+    {
+      // "NAME": "BAT",
+      cToken: "0x6C8c6b02E7b2BE14d4fA6022Dfd6d75921D90E4E",
+      underlying: "0x0D8775F648430679A709E98d2b0Cb6250d2887EF",
+      symbolHash: "0x3ec6762bdf44eb044276fec7d12c1bb640cb139cfd533f93eeebba5414f5db55",
+      baseUnit: "1000000000000000000",
+      priceSource: "2",
+      fixedPrice: "0",
+      uniswapMarket: "0xae614a7a56cb79c04df2aeba6f5dab80a39ca78e",
+      reporter: "0xeBa6F33730B9751a8BA0b18d9C256093E82f6bC2",
+      reporterMultiplier: "10000000000000000",
+      isUniswapReversed: false
+    },
+    {
+      // "NAME": "ZRX",
+      cToken: "0xB3319f5D18Bc0D84dD1b4825Dcde5d5f7266d407",
+      underlying: "0xE41d2489571d322189246DaFA5ebDe1F4699F498",
+      symbolHash: "0xb8612e326dd19fc983e73ae3bc23fa1c78a3e01478574fa7ceb5b57e589dcebd",
+      baseUnit: "1000000000000000000",
+      priceSource: "2",
+      fixedPrice: "0",
+      uniswapMarket: "0x14424eeecbff345b38187d0b8b749e56faa68539",
+      reporter: "0x5c5db112c98dbe5977A4c37AD33F8a4c9ebd5575",
+      reporterMultiplier: "10000000000000000",
+      isUniswapReversed: true
+    },
+    {
+      // "NAME": "REP",
+      // Warning: as of 2021-09-13, this has very low liquidity ($33.6k)
+      cToken: "0x158079Ee67Fce2f58472A96584A73C7Ab9AC95c1",
+      underlying: "0x1985365e9f78359a9B6AD760e32412f4a445E862",
+      symbolHash: "0x91a08135082b0a28b4ad8ecc7749a009e0408743a9d1cdf34dd6a58d60ee9504",
+      baseUnit: "1000000000000000000",
+      priceSource: "2",
+      fixedPrice: "0",
+      uniswapMarket: "0xb055103b7633b61518cd806d95beeb2d4cd217e7",
```

Comment:
https://info.uniswap.org/#/pools/0xb055103b7633b61518cd806d95beeb2d4cd217e7

Reply:
Indeed, I don't see it on their interface.

Reply:
Can/should we remove it then?

Reply:
We raised it with Compound folks. They said the market has been deprecated and at some point the protocol hopes to eject it from the system, but they haven't gotten to that point yet.